File: old/python/door_status.py
import time
import RPi.GPIO as io
io.setmode(io.BCM)

door_pin = 5

io.setup(door_pin, io.IN, pull_up_down=io.PUD_UP)

while True:
    if io.input(door_pin):
        print("Kegerator Door Open")
    else:
        print("Kegerator Door Closed")
    time.sleep(0.5)

File: door_status.rb
require 'rpi_gpio'

RPi::GPIO.set_numbering :bcm

PIN_NUM = 5

RPi::GPIO.setup PIN_NUM, as: :input, pull: :up

loop do
  if RPi::GPIO.high? PIN_NUM
    puts('Kegerator door is open! Close it!')
  else
    puts('Kegerator door is closed.')
  end

  sleep 2.0
end
